readthedocs/rtd_tests/tests/test_core_tags.py

```diff
@@ -9,10 +11,26 @@
 from readthedocs.core.templatetags import core_tags
 
 
-@override_settings(USE_SUBDOMAIN=False, PUBLIC_DOMAIN='readthedocs.org')
+@override_settings(USE_SUBDOMAIN=False, PRODUCTION_DOMAIN='readthedocs.org')
 class CoreTagsTests(TestCase):
     fixtures = ["eric", "test_data"]
 
+    # Determine if we are under community (http) or corporate site (https)
+    scheme = 'https' if settings.CELERY_APP_NAME == 'readthedocsinc' else 'http'
```

Comment:
I'm using `conftest.py` to set these options. Example,

```python

PYTEST_OPTIONS = (
    # Include `not community` to parameters so that community tests do not run
    ('markexpr', 'not community'),

    # Options to set test environment
    ('community', False),
    ('environment', 'readthedocsinc'),

    ('url_scheme', 'https'),
)


def pytest_configure(config):
    for option, value in PYTEST_OPTIONS:
        setattr(config.option, option, value)
```

So, this way in the test case I'm doing `pytest.config.option.url_scheme` which will be `https` in the corporate site and `http` in the community.

This way, we will have all the test options under this file and easy to tweak/modify/understand where they come from.